sympy/core/sympify.py

```diff
@@ -257,6 +262,20 @@ def sympify(a, locals=None, convert_xor=True, strict=False, rational=False,
     The keywords ``rational`` and ``convert_xor`` are only used
     when the input is a string.
 
+    convert_xor
+    -----------
+    >>> sympify('x^y',convert_xor=True)
```

Comment:
There should be a blank line before the example

Reply:
yes